package.json

```diff
@@ -204,7 +207,7 @@
 		"lint-js": "wp-scripts lint-js",
 		"lint-js:fix": "npm run lint-js -- --fix",
 		"lint-php": "wp-scripts env lint-php",
-		"lint-pkg-json": "wp-scripts lint-pkg-json ./packages",
+		"lint-pkg-json": "wp-scripts lint-pkg-json 'packages/*/package.json'",
```

Comment:
This also looks like a breaking change. @tclindner - was it intended?

Reply:
Hi @gziolo, no it was not. I've released v4.0.3 that should correct the issue. Please let me know if you have other issues!
https://github.com/tclindner/npm-package-json-lint/releases/tag/v4.0.3

Reply:
Nice, thanks. We will update the dependency right away 👍

Reply:
> Nice, thanks. We will update the dependency right away 👍

We are using ranges so we probably should be fine, @ntwb @nerrad or @swissspidy can you confirm?

Reply:
The _root_ lock file needs updating for this, currently:
```json
24344          "npm-package-json-lint": {
24345                  "version": "4.0.2",
...
```

Reply:
I opened https://github.com/WordPress/gutenberg/pull/18160 to address it.